src/components/SingleReview.jsx

```diff
@@ -30,50 +32,18 @@ function SingleReview() {
 
     if (comments.length === 0) {
         return (
-            <>
-            <section className="review">
-                <h2>{review.title}</h2>
-                <img className="reviewImg"src={review.review_img_url} alt={review.title}/>
-                <p className="reviewBody">{review.review_body}</p>
-                <h3>Additional Information: </h3>
-                <ul className="reviewInfo">
-                    <li> Owner: {review.owner}</li>
-                    <li> Designer: {review.designer}</li>
-                    <li> Category: {review.category}</li>
-                    <li> Votes: {review.votes}</li>
-                </ul>
-            </section>
-            <section className="commentsClass">
-            <h3 className="commentHeader">Comments </h3>
-            <p>No comments have been posted on this thread</p>
-            </section>
-            </>
-            )
+        <>
+        <RenderReviewInformation review={review}/>
+        <RenderNoComments/>
```

Comment:
At the moment while comments are loading we get "no comments have been posted...". We should add a different feedback to the users that are waiting for the comments to load 🙂